var/spack/repos/builtin/packages/qthreads/package.py

```diff
@@ -58,13 +62,15 @@ class Qthreads(AutotoolsPackage):
     variant("static", default=True, description="Build static library")
     variant(
         "stack_size",
-        default=4096,
+        default=16384,
         description="Specify number of bytes to use in a stack",
         values=is_integer,
     )
 
     depends_on("hwloc@1.0:1", when="@:1.15 +hwloc")
-    depends_on("hwloc@1.5:2", when="@1.16: +hwloc")
+    depends_on("hwloc@1.5:2", when="@:1.16 +hwloc")
```

Comment:
Is this:
```suggestion
    depends_on("hwloc@1.5:2", when="@1.16 +hwloc")
```
what you meant?